aspnetcore/release-notes/aspnetcore-3.0.md

```diff
@@ -134,7 +134,184 @@ For more information, see <xref:blazor/index>.
 
 ## SignalR
 
-@bradygaster  to provide
+The SignalR JavaScript client has changed from being `@aspnet/signalr` to `@microsoft/signalr`. To react to this change, you will need to change your references in `package.json` files, require statements, and ECMAScript import statements. 
+
+In the JavaScript and .NET Clients for SignalR, we've added support for automatic reconnection via the `withAutomaticReconnect()` method on the `HubConnectionBuilder`. By default, the client will try to reconnect immediately and after 2, 10, and 30 seconds. Enlisting in automatic reconnect is opt-in, but simple via this new method.
```

Comment:
It might be nice to call out that this will give you a new ConnectionId which is different from the OG SignalR clients. Maybe something like the following would be good.

```suggestion
In the JavaScript and .NET Clients for SignalR, we've added support for automatic reconnection via the `withAutomaticReconnect()` method on the `HubConnectionBuilder`. By default, the client will try to reconnect immediately and retry after 2, 10, and 30 seconds if necessary. If the client successfully reconnects, it will be given a new connection ID. Enlisting in automatic reconnect is opt-in, but simple via this new method.
```

Reply:
@halter73 I'll migrate your suggestion over to #14371